### ISSUE TITLE
no-indistinguishable-colors problems with shadows and inherited property values

### ISSUE THREAD

**Issue by Sigiller** [stylelint/stylelint#1495]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

1. There is should be an option which force this rule evaluate only colors of explicitly defined properties, but not inherited.
2. Warnings about indistinguishability of text color and text shadow or box shadow is unexpected. 

> Which rule, if any, is this issue related to?

`no-indistinguishable-colors `

> What CSS is needed to reproduce this issue?

```css
.foo {
  color: black;
}

.foo .child {
   box-shadow: 1px 1px 1px rgba(0, 0, 0, 0.5);
   text-shadow: 1px 1px 1px rgba(0, 0, 0, 0.5);
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
      "no-indistinguishable-colors": true
  }
}
```

> Which version of stylelint are you using?

` 6.6.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI with `stylelint --config myconfig *.css`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged:
```
 6:28  ×  Unexpected indistinguishable colors "rgba(0, 0, 0, 0.5)" and "black"   no-indistinguishable-colors

 7:29  ×  Unexpected indistinguishable colors "rgba(0, 0, 0, 0.5)" and "black"   no-indistinguishable-colors
```

**Comment by jeddy3** [comment]
@Sigiller Thanks for the detailed write-up.

As per this rule's [README](https://github.com/stylelint/stylelint/blob/master/src/rules/no-indistinguishable-colors/README.md) though, feature requests need to be directed towards to the [colorguard issue tracker](https://github.com/SlexAxton/css-colorguard/issues) as this rule simple wraps that module.

**Comment by Sigiller** [comment]
@jeddy3 Ok, thanks.
